install-swagger-ui.sh

```diff
@@ -0,0 +1,8 @@
+# Copyright 2015, EMC, Inc.
+
+cd ./static
+rm -rf swagger-ui
+curl -L -o swagger-ui.zip https://github.com/zyoung51/swagger-ui/archive/master.zip
```

Comment:
Looks like @heckj just forked swagger-ui for that.

Yeah, I think it makes sense to fork as a pattern for upstream dependencies we want to host ourselves, we do it with DI, web-ui as you mention, swagger-ui now, lib-ffi, etc. It can be a pain if you have to keep them in sync but seems like an okay pattern for just grabbing assets. Of course, it still means we're tied to Github, but you have to draw the line somewhere :)